src/lib.rs

```diff
@@ -265,7 +265,7 @@ pub enum LineNumbers {
 }
 
 impl LineNumbers {
-    const EXTRA_PADDING: u8 = 5;
+    const EXTRA_PADDING: usize = 5;
```

Comment:
Any reason why you changed the u8 to a usize. I think `u8` is better here as it is a lot smaller and is pretty efficient for storing 5 (b00000101). Also it is more descriptive for this situation.

Reply:
I made this change since EXTRA_PADDING is only being used to add with a usize, and so it has to be converted to a usize anyways. A u8 is smaller, but I don't think the 15 bytes that it saves will matter too much (or even at all, since there may be alignment padding added so that it effectively takes up the same space anyways).